lib/foreman_maintain/concerns/primary_checks.rb

```diff
@@ -0,0 +1,24 @@
+module ForemanMaintain
+  module Concerns
+    module PrimaryChecks
+      def validate_downstream_packages
+        package = package_name
+        if !package.nil? && detector.feature(:installer).with_scenarios?
+          unless package_manager.installed?(package)
+            raise StandardError, "Error: Important rpm package #{package} is not installed!"\
+                  "\nInstall #{package} rpm to ensure system consistency."
+          end
+        end
+      end
+
+      def package_name
+        installed_scenario = detector.feature(:installer).last_scenario
+        if installed_scenario == 'satellite'
+          'satellite'
```

Comment:
Features `capsule` & `satellite` have `package_name` method but those are not declared as class methods. 
If those will be class methods then I guess we can access them here and no need to hardcode them at multiple places.
BTW, this is just a thought so I don't want to block the PR due this change.

Reply:
Modified to use class methods.